cvdupdate/__main__.py

```diff
@@ -209,6 +209,7 @@ def serve(port: int, config: str, verbose: bool):
     m = CVDUpdate(config=config, verbose=verbose)
     os.chdir(str(m.db_dir))
     m.logger.info(f"Serving up {m.db_dir} on localhost:{port}...")
+    auto_updater.start()
```

Comment:
That changes the default behaviour, which reduces the chance of being accepted to the upstream repo.

I suggest an option `--update-interval-seconds` with the default being 0 (feature disabled)